src/styles/types.js

```diff
@@ -483,11 +483,25 @@ export type BordersT = {
   border400: BorderT,
   border500: BorderT,
   border600: BorderT,
-  useRoundedCorners: boolean,
   radius100: string,
   radius200: string,
   radius300: string,
   radius400: string,
+  buttonBorderRadius: string,
+  cardBorderRadius: string,
+  checkboxBorderRadius: string,
+  toggleOuterBorderRadius: string,
+  toggleInnerBorderRadius: string,
+  datepickerBorderRadius: string,
+  datepickerInteractionBorderRadius: string,
+  inputBorderRadius: string,
+  menuBorderRadius: string,
+  toastBorderRadius: string,
+  progressBarBorderRadius: string,
+  sliderBorderRadius: string,
+  sliderThumbBorderRadius: string,
+  tableBorderRadius: string,
+  tagBorderRadius: string,
```

Comment:
What about popover corners?

Reply:
Where are do the popover border radius styles get applied? They weren't in the popover or toast directories 🤔

Reply:
Ah - the popover does not have border radius styles by default right? I added a `toastBorderRadius` for the toast though.

Reply:
It has a `boxShadow` that shapes the corners. So we need to add border radius for the rounded corners case.

Reply:
I don't understand - how can the boxShadow shape the corners?

Reply:
Here is an example to help understand the issue: https://codesandbox.io/embed/popover-opens-on-click-4m55s

Reply:
An example above is just to showcase the discussed issue. A real world example would be `Menu` in a `Popover` when rounded corners set to be used.